data/plugins/treeview.lua

```diff
@@ -28,13 +32,17 @@ local tooltip_alpha_rate = 1
 
 
 local function get_depth(filename)
+  if filename == "" then return 0 end
   local n = 1
-  for _ in filename:gmatch(PATHSEP) do
-    n = n + 1
+  for sep in filename:gmatch("\\*" .. PATHSEP) do
```

Comment:
In what cases is this a problem?
Shouldn't paths already be somewhat normalized when they get here?

Reply:
I would assume so, yeah. Truthfully, it's been too long since I've looked at it.

Apparently this was an issue on Linux according to the commit message. I just tried it, after reverting, and it seems fine, so I guess I'll revert this.